zulipterminal/config/keys.py

```diff
@@ -136,6 +136,11 @@
         'keys': {'i'},
         'help_text': 'View message information',
     }),
+    ('EDIT_HISTORY', {
+        'keys': {'e'},
+        'help_text': 'View edit history',
+        'excluded_from_random_tips': True,
+    }),
```

Comment:
We may have to change this key later, if we do plan to use this key for editing messages.